beta/src/pages/learn/start-a-new-react-project.md

```diff
@@ -1,41 +1,41 @@
 ---
-title: Start a New React Project
+title: एक नया React प्रोजेक्ट शुरू करें
 ---
 
 <Intro>
 
-If you're learning React or considering adding it to an existing project, you can get started quickly by [adding React to any HTML page with script tags](/learn/add-react-to-a-website). If your project will need many components and many files, it might be time to consider the options below!
+यदि आप React सीख रहे हैं या इसे किसी मौजूदा प्रोजेक्ट में इस्तेमाल करने पर विचार कर रहे हैं, तो आप [स्क्रिप्ट टैग को किसी भी HTML पेज पर React इस्तेमाल करके](/learn/add-react-to-a-website) शुरू कर सकते हैं। यदि आपके प्रोजेक्ट को कई कौम्पोनॅन्ट और कई फाइलों की आवश्यकता होगी, तो यह नीचे दिए गए औपशंस पर विचार करने का समय हो सकता है!
 
 </Intro>
 
-## Choose your own adventure {/*choose-your-own-adventure*/}
+## अपना खुद का रोमांच चुनें {/*अपना खुद का रोमांच चुनें*/}
```

Comment:
Lets not change the text written like this `{/*choose-your-own-adventure*/}`
These are used for reflinks

`{/*अपना खुद का रोमांच चुनें*/}` = `{/*choose-your-own-adventure*/}`

Reply:
@NegiAkash890 I think my comment was not so clear here. We actually want to translate the title here but do not want to translate the text in the brackets. So this should be changed something like this

```
## अपना खुद का रोमांच चुनें {/*choose-your-own-adventure*/}
```